nltk/translate/api.py

```diff
@@ -20,16 +20,17 @@ class AlignedSent(object):
     Return an aligned sentence object, which encapsulates two sentences
     along with an ``Alignment`` between them.
 
-        >>> from nltk.translate import AlignedSent
+        >>> from nltk.translate import AlignedSent, Alignment
+        >>> from nltk.metrics import precision
         >>> algnsent = AlignedSent(['klein', 'ist', 'das', 'Haus'],
-        ...     ['the', 'house', 'is', 'small'], '0-2 1-3 2-1 3-0')
+        ...     ['the', 'house', 'is', 'small'], Alignment.fromstring('0-2 1-3 2-1 3-0'))
         >>> algnsent.words
         ['klein', 'ist', 'das', 'Haus']
         >>> algnsent.mots
         ['the', 'house', 'is', 'small']
         >>> algnsent.alignment
         Alignment([(0, 2), (1, 3), (2, 1), (3, 0)])
-        >>> algnsent.precision('0-2 1-3 2-1 3-3')
+        >>> precision(Alignment.fromstring('0-2 1-3 2-1 3-3'), algnsent.alignment)
```

Comment:
When `precision` is removed from `AlignedSent`, this doctest should be removed. Example integrations with `nltk.metrics` and other modules could be shown in `test\translate.doctest`.